account-kit/infra/src/chains.ts

```diff
@@ -236,3 +236,39 @@ export const zoraSepolia: Chain = {
     ...vzoras.rpcUrls,
   },
 };
+
+export const worldChainSepolia: Chain = defineChain({
```

Comment:
nit: either make this `worldchainSepolia` or make the one below `worldChain`

Reply:
made it worldChain